src/entities/euring-codes/ringing-scheme-entity.ts

```diff
@@ -3,9 +3,16 @@ import { Length, IsAlpha, IsOptional, IsString } from 'class-validator';
 import { equalLength } from '../../validation/validation-messages';
 import { Ring } from '../ring-entity';
 
+export interface RingingSchemeDto {
```

Comment:
please extend `EntitiyDto`. Do it the way you used to do it)

Reply:
I can't do this since RingingSchemeDto doesn't have description